### DISCUSSION TITLE
M5atom documentation?

### DISCUSSION THREAD

**Discussion by 3dprintedmac** [josephdadams/TallyArbiter#468] [Q&A]
Hi. So I’m new to using m5 controllers. I’ve done some programming stuff with raspberry pi to function with my 3-D printers and have walked through tweaking marlin with VSCode, but that’s about the extent of my current knowledge.

does anyone have a rough step-by-step on how to set up the M5 atom with tailbat and 5x5matrix for tally arbiter? I’d usually go with the Pico direction for the attempt, but I can’t find that pico zero W anywhere. I’m looking for the easiest cheap way to set up the more bright Tally lights that’s not making my guys utilize their phones anymore. 

I’m just not adept enough to take the current instruction video and know how to tweak for use with the different hardware without someone helping point me in the right direction of what key parts in the code I’d need to adjust. 

Or if you could link where someone else has walked through the M5atom setup already, or if you know where to find pico zero W’s to go that route, or whatever might get me where I can have partial confidence that I can tinker my way to a working piece of hardware haha. I’m not sure I could currently utilize just the M5 stick-c in my current setup unless it’s basically just to get familiar with the process since it seems like the screen is fairly dim and is not the best choice for wireless option. 

Any help would be appreciated, thanks!

**Comment by josephdadams** [comment]
The tailbat is just a battery. There are some M5 atom walkthrough videos with Tally Arbiter on Youtube.

**Comment by josephdadams** [comment]
https://www.youtube.com/watch?v=soj1Cxv3mLY
